pgen-native/Python.h

```diff
@@ -0,0 +1,56 @@
+#ifndef __PYTHON_H__
+#define __PYTHON_H__
+
+/**
+ * Stub out the bits of python on which which parser.c depends.
+ *
+ */
+
+#include <assert.h>
+#include <ctype.h>
+#include <limits.h>
+#include <stdbool.h>
+#include <stddef.h>
+// XXX: Remove instances of these later
+#define PyAPI_FUNC(x) extern x
+#define PyAPI_DATA(x) extern x
+
+#define PY_SIZE_MAX SSIZE_MAX
+
+// Define this, because it's true.
+#define PGEN
+
+// Types
+typedef size_t Py_ssize_t;
```

Comment:
Woops, this should be `ssize_t`. Fix that.

Then I would like to replace all instances of `Py_ssize_t` with `ssize_t`.